base/applications/rapps/appview.cpp

```diff
@@ -1299,7 +1299,7 @@ CAppsListView::AddApplication(CAppInfo *AppInfo, BOOL InitialCheckState)
             hIcon = ExtractIconW(hInst, szIconPath.GetString(), 0);
         }
 
-        if (!hIcon)
+        if (!hIcon || (hIcon == (HICON)(INT_PTR)1))
```

Comment:
```suggestion
        if (hIcon == (HICON)1 || !hIcon)
```
Meanwhile, this change may be wrong as is:
Isn't `LoadIconW()` going to fail too, in the new case?

Reply:
@SergeGautherie this reversing doesn't really help.

Reply:
>  Isn't LoadIconW() going to fail too, in the new case?

Which LoadIconW() call? The one commented as "default icon"?

Reply:
> this reversing doesn't really help.

Only seems a bit more logical: first no file then no icons.

> Which LoadIconW() call? The one commented as "default icon"?

Removed : I misread.